src/smolagents/models.py

```diff
@@ -1610,6 +1616,11 @@ def generate(
         )
 
 
+class OpenAIModel(OpenAIServerModel):
+    def __new__(cls, *args, **kwargs):
+        return super().__new__(cls)
```

Comment:
Is this just an alias or are you planning to deprecate `OpenAIServerModel`?

If this is just an alias and both are identically valid, then I would suggest:
```python
OpenAIModel = OpenAIServerModel
```

If you are planning to deprecate `OpenAIServerModel`, then you should inherit inversely:
```python
class OpenAIServerModel(OpenAIModel):
    def __new__(cls, *args, **kwargs):
        warnings.warn(
            "OpenAIServerModel was renamed to OpenAIModel in version 1.19.0 and will be removed in 1.22.0. "
            "Please use OpenAIModel instead.",
            FutureWarning,
            stacklevel=2
        )
        return super().__new__(cls)
```